src/main/java/org/folio/inventory/validation/ItemStatusValidator.java

```diff
@@ -0,0 +1,38 @@
+package org.folio.inventory.validation;
+
+
+import java.util.Optional;
+
+import org.apache.commons.lang3.StringUtils;
+import org.folio.inventory.domain.items.ItemStatusName;
+import org.folio.inventory.support.JsonHelper;
+import org.folio.inventory.support.http.server.ValidationError;
+
+import io.vertx.core.json.JsonObject;
+
+public final class ItemStatusValidator {
+
+  private ItemStatusValidator() {}
+
+  public static Optional<ValidationError> itemHasCorrectStatus(JsonObject itemRequest) {
+    final String statusName = JsonHelper.getNestedProperty(itemRequest, "status", "name");
+
+    if (StringUtils.isBlank(statusName)) {
+      return Optional.of(new ValidationError(
```

Comment:
Is this formatting mimicking code I’d previously written, or do you find the parameters easier to interpret with this formatting?

Reply:
I've changed formatting, now all the parameters on the single line.

Is it something you meant?